tests/integration-tests/tests/health_check/test_scheduled_events/test_scheduled_events/disable_health_check.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+# Disable health check in sqswatcher config
+sudo bash -c "sed -i '/disable_health_check.*/d' /etc/sqswatcher.cfg"
+sudo bash -c "sed -i '\$adisable_health_check = True' /etc/sqswatcher.cfg"
+
+# Restart sqswatcher
+sudo bash -c "source /usr/local/pyenv/versions/cookbook_virtualenv/bin/activate && supervisorctl restart sqswatcher"
```

Comment:
minor: no need to activate `/usr/local/pyenv/versions/cookbook_virtualenv/bin/supervisorctl` should work

Reply:
Done